Assets/VRTK/Source/Scripts/Interactions/Interactables/VRTK_InteractableObject.cs

```diff
@@ -198,6 +200,12 @@ public enum ValidDropTypes
         [Tooltip("Determines which controller can initiate a use action.")]
         public AllowedController allowedUseControllers = AllowedController.Both;
 
+        [Header("Stack Settings")]
+        [Tooltip("Determines if the object can be grouped with identical objects within a `VRTK_SnapDropZone`.")]
+        public bool isStackable = false;
+        [Tooltip("Used to determine whether objects are the same and therefore stackable within a `VRTK_SnapDropZone`.")]
+        public string objectID = "defaultInteractableObjectID";
```

Comment:
wouldn't a policy list be better here rather than a rigid string?

Then you could determine things to be the same based on tag/layer/script applied.

Also, I don't like that this extra data is needed to be added to the Interactable Object. I feel it should be in a separate script so the Interactable Object doesnt get fatter than what it already is.